### PR TITLE
[BPK-4447] Changed stars to use darker colours

### PR DESCRIPTION
**Pull request by shaundon**

### CODE REVIEW

**Review by olliecurtis — COMMENTED** [review]
LGTM - will check it out once deployed and then re approve

**Review by georgegillams — APPROVED** [review]

**Review by olliecurtis — APPROVED** [review]
@shaundon has shown me a screenshot as the PR is still caching the build so can't view it right now

**Pull request merged**
